include/exadg/postprocessor/output_generator_scalar.cpp

```diff
@@ -74,8 +72,14 @@ OutputGenerator<dim, Number>::setup(dealii::DoFHandler<dim> const & dof_handler_
   mapping     = &mapping_in;
   output_data = output_data_in;
 
-  // reset output counter
-  output_counter = output_data.start_counter;
+  time_control.reinit(output_data_in.write_output,
+                      output_data_in.start_time,
+                      std::numeric_limits<double>::max(), // TODO: currently placeholder
+                      output_data_in.interval_time,
+                      dealii::numbers::invalid_unsigned_int); // TODO: currently placeholder
```

Comment:
I think here we should also work with `numeric_limits::max()`. However, I am not sure because we have to make sure that the asserts in `TimeControl::needs_evaluation()` do not cause problems.

Reply:
should be done